examples/standalone/optimiser.py

```diff
@@ -1,7 +1,7 @@
 import numpy as np
 from scipy.optimize import minimize
 
-from pybop import BaseOptimiser
+from pybop.optimisers.base_optimiser import BaseOptimiser, Result
```

Comment:
I think we should add `Result` to `pybop/__init__.py` so the class is accessible throughout the library. This allows us to simply import it as `from pybop import Result` here and below.